examples/dao/setup_sigma_dao.sql

```diff
@@ -5,9 +5,10 @@
 -- create a sigma_daos table if does not exit already.
 CREATE TABLE IF NOT EXISTS sigma_daos (
   id SERIAL PRIMARY KEY, -- auto increment id
-  app_id bigint, -- application id
-  app_params jsonb, -- application params with approval etc.
-  asset_id bigint -- token id
+  app_id BIGINT, -- application id
+  dao_name CHAR(255), -- dao name
+  app_params JSONB, -- application params with approval etc.
+  asset_id BIGINT -- token id
```

Comment:
why are we moving to Caps here?

Reply:
I prefer to keep sql keywords in caps. This is also recommended by postgres as well.